src/app/page.tsx

```
@@ -1,55 +1,229 @@
+          <nav className="hidden md:flex items-center gap-8 text-sm font-medium">
+            <Link href="#features" className="text-white/70 hover:text-white transition-colors">Features</Link>
+            <Link href="#models" className="text-white/70 hover:text-white transition-colors">Models</Link>
+            <Link href="#about" className="text-white/70 hover:text-white transition-colors">About</Link>
+            {session ? (
+              <Link href="/dashboard" className="bg-[#a259ff] hover:bg-[#a259ff]/90 text-white px-5 py-2 rounded-md transition-colors">Dashboard</Link>
+            ) : (
+              <Link href="/api/auth/signin" className="bg-[#a259ff] hover:bg-[#a259ff]/90 text-white px-5 py-2 rounded-md transition-colors">Sign In</Link>
+            )}
+          </nav>
+          <button className="md:hidden text-white" aria-label="Open menu">
+            <svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24" fill="none" stroke="currentColor" strokeWidth="2" strokeLinecap="round" strokeLinejoin="round">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)